### PR TITLE
Update bulk cve update

### PR DESCRIPTION
**Pull request by albertkol**
## Done

- Releases no longer needed at all, since we do not create releases through this endpoint
- Only update cves/statuses/packages if needed. We use a bool for each to find out if there are any changes coming from the data, if not do not update anything
- Delete only `cve.statuses` that we do not find in the cve data we receive; before we were deleting everything the inserting again which was too heavy and results in time outs. 

From my tests:
```
Before:
1st run - Insert round - Time: 7m0.526s
2nd run - Update round - Time: 7m32.567s

After:
1st run - Insert round - Time: 6m36.694s
2nd run - Update round - Time: 5m50.945s
```

### CODE REVIEW

**Comment by carkod** [comment]
QA? is this an endpoint?

File: webapp/security/views.py [hunk 1]
```diff
@@ -607,6 +607,36 @@ def update_statuses(cve, data, packages, releases):
...
+def delete_statuses(cve, data):
+    statuses_to_delete = (
+        db_session.query(Status).filter(Status.cve_id == cve.id).all()
+    )
+    statuses = cve.packages
+
+    for package_data in data.get("packages", []):
+        name = package_data["name"]
+
+        if not statuses[name]:
```
Comment (jpmartinspt) [inline webapp/security/views.py:619]: Won't this result in a `KeyError` if the key is not there? I think you want `if name in statuses`?

Comment (albertkol) [reply webapp/security/views.py:619]: You are correct. My mistake.

File: webapp/security/views.py [hunk 2]
```diff
@@ -571,40 +571,67 @@ def cve(cve_id):
...
 
-            status.status = status_data["status"]
-            status.description = status_data["description"]
+            if "component" not in status_data:
+                status_data["component"] = None
+            if "pocket" not in status_data:
+                status_data["pocket"] = None
 
-            if "component" in status_data:
+            if status.component != status_data["component"]:
```
Comment (jpmartinspt) [inline webapp/security/views.py:621]: You could remove the if statments above by doing `status_data.get("component")`, right?

Comment (albertkol) [reply webapp/security/views.py]: Done.

**Review by jpmartinspt — APPROVED** [review]

**Comment by 5325833510** [comment]
Fatma

**Pull request merged**